packages/main/src/requests/stream-reader.ts

```diff
@@ -75,9 +75,27 @@ export function processStream(response: Response): GenerateContentStreamResult {
  * GenerateContentResponse in each iteration.
  */
 function readFromReader(
-  reader: ReadableStreamDefaultReader,
+  bodyReader: ReadableStreamDefaultReader,
 ): ReadableStream<GenerateContentResponse> {
   let currentText = "";
+  const reader = new ReadableStream<Uint8Array>({
```

Comment:
Since `response.body` is a ReadableStream<Uint8Array> I think we can just pass `body` to this function and here we can do `body.pipeThrough(newTextDecoderStream()).getReader()`. But for the tests we'll have to rewrite mockResponse to really make `body` a ReadableStream instead of an empty object with a getReader() function

Reply:
And it's not working in Node.js v21.2.0 - fetch body request have only one `getReader` method :(